test/cdk-test-integration/lib/cdk-test-integration-stack.ts

```diff
@@ -0,0 +1,16 @@
+import { Stack, StackProps } from 'aws-cdk-lib';
+import * as ddb from 'aws-cdk-lib/aws-dynamodb';
+import { Construct } from 'constructs';
+import { GlobalTable } from '../../../lib/GlobalTable';
```

Comment:
Make sure to compile and test your code (`yarn compile` and/or `yarn build`) between every PR revision. This line currently does not compile because the `../../../lib` directory does not exist. Your import should be:

```suggestion
import { GlobalTable } from '../../../src';
```

Notice I also removed the `GlobalTable` part at the end for two reasons:
1. There is no longer a file called `GlobalTable.ts` 
2. You don't need to include the specific filename at the end of this import statement because the `GlobalTable` API is exported from the `src` directory. 

If you are making a change which changes the generated cloudformation template, you should also fully deploy this integration test to make sure that your change is working.

Reply:
You can also check this from the PR build, it was caught there as well.